src/Tasks/WriteCodeFragment.cs

```diff
@@ -44,6 +48,17 @@ public class WriteCodeFragment : TaskExtension
         [Required]
         public string Language { get; set; }
 
+        /// <summary>
+        /// If the projects have ImplictUsings set, then use it's value.
+        /// Anything other than "enable" changes nothing here.
+        /// When it is set to "enable" then do not add the usings that
+        /// this normally writes to the output file.
+        /// This leverages the full power of what was added in .NET 6+
+        /// called global usings of C# 10 which is specific to C# (I think).
+        /// </summary>
+        [Required]
+        public string ImplicitUsings { get; set; }
```

Comment:
Because this breaks the .NET SDK itself, should I default this to ``disable`` and mark it as not required until the SDK updates to start using it for the task always (and mark it as required in a follow-up pr?